_posts/cleaning-up-your-test-data-builders.md

```diff
@@ -0,0 +1,104 @@
+---
```

Comment:
I really reading the entire article. It really gets to the point and the end result really looks like an easy to use solution!